src/sources/docker_logs/mod.rs

```diff
@@ -976,10 +1120,19 @@ impl ContainerLogInfo {
                 // Otherwise, create a new partial event merge state with the
                 // current message being the initial one.
                 if let Some(partial_event_merge_state) = partial_event_merge_state {
-                    partial_event_merge_state
-                        .merge_in_next_event(log_event, &[log_schema().message_key().to_string()]);
+                    match log_namespace {
+                        LogNamespace::Vector => {
+                            partial_event_merge_state.merge_in_next_event(log, &["."]);
```

Comment:
:thinking:  just wondering if this line and the other like it might benefit from a quick comment to ease unfamiliar eyes.

Reply:
[6ae5da3](https://github.com/vectordotdev/vector/pull/15416/commits/6ae5da3b31d5ccf0acbaf932e0ee0a1cbfb86d70) added a quick comment